### DISCUSSION TITLE
"Load Film Model" and "Film Interpolation" nodes not working correctly?

### DISCUSSION THREAD

**Discussion by RuschMeyer** [melMass/comfy_mtb#149] [Q&A]
<img width="393" alt="Bildschirmfoto 2024-02-06 um 09 46 37" src="https://github.com/melMass/comfy_mtb/assets/130401652/073a991c-d934-4386-a869-d72ec5133521">

I have the mtb custome nodes installed and still missing those two nodes. Tried on PC, Mac and RunDiffusion. Is everything alright with those nodes?

**Comment by joshuachung** [comment]
Have you get it fixed? 

I am facing the similar issue.

![Screenshot 2025-05-01 172629](https://github.com/user-attachments/assets/1c503cb1-3dd4-44dc-9c8c-e3d6a1a13d49)
